code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -184,3 +184,31 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/Madvilliansmask
+	name = "the MadVillian's mask"
+	desc = "A metal mask formerly owned by the Madvillian."
+	icon_state = "madvilliansmask"
+	inhand_icon_state = "madvilliansmask"
+	w_class = WEIGHT_CLASS_SMALL
+	modifies_speech = TRUE
+
+/obj/item/clothing/mask/Madvilliansmask/dropped(mob/M)
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
```

Comment:
I don't understand this, you're not registering any signals.

Reply:
UnregisterSignal(M, COMSIG_MOB_SAY) doesn't need to register a signal, It just makes it so when you remove the mask the user stops using DOOMspeak

Reply:
you dont register a signal you mouthbreather